MapboxNavigation/NavigationViewController.swift

```diff
@@ -258,18 +258,22 @@ public class NavigationViewController: UIViewController, RouteMapViewControllerD
     @objc public var sendsNotifications: Bool = true
     
     /**
-     Shows a button that allows drivers to report feedback such as accidents, closed roads,  poor instructions, etc. Defaults to `false`.
+     Shows a button that allows drivers to report feedback such as accidents, closed roads,  poor instructions, etc. Defaults to `true`.
      */
     @objc public var showsReportFeedback: Bool = true {
         didSet {
             mapViewController?.reportButton.isHidden = !showsReportFeedback
-            showsEndOfRoute = showsReportFeedback
+            showsEndOfRouteFeedback = showsReportFeedback
         }
     }
     
-    @objc public var showsEndOfRoute: Bool = true {
+    
+    /**
+    Shows End of route Feedback UI when the route controller arrives at the final destination. Defaults to `true.`
+    */
+    @objc public var showsEndOfRouteFeedback: Bool = true {
```

Comment:
This property name makes it sound like it shows feedback that has already been collected. `asksForFeedbackOnArrival` or `showsArrivalFeedbackPrompt` would be more accurate.